src/main/kotlin/apply/application/ApplicationFormService.kt

```diff
@@ -18,44 +18,44 @@ class ApplicationFormService(
     private val applicationFormRepository: ApplicationFormRepository,
     private val recruitmentRepository: RecruitmentRepository,
     private val recruitmentItemRepository: RecruitmentItemRepository,
-    private val applicationValidator: ApplicationValidator
+    private val userValidator: UserValidator
 ) {
-    fun create(applicantId: Long, request: CreateApplicationFormRequest) {
+    fun create(userId: Long, request: CreateApplicationFormRequest) {
         val recruitment = findApplicableRecruitment(request.recruitmentId)
-        check(!applicationFormRepository.existsByRecruitmentIdAndApplicantId(recruitment.id, applicantId)) {
+        check(!applicationFormRepository.existsByRecruitmentIdAndUserId(recruitment.id, userId)) {
```

Comment:
개인적으로 부정에 부정이라서 코드를 이해하기가 조금 어려웠어요 😅
ApplicationFormRepository의 `existsByRecruitmentIdAndApplicantId`를 위의 함수로 변경해보면 어떨까요? 
```suggestion
        check(!applicationFormRepository.notExistsByRecruitmentIdAndUserId(recruitment.id, userId)) {
```

또한 해당 코드에 대한 테스트 코드가 없는 것 같은데  아래의 테스트 코드를 추가하면 좋을 것 같아요! (메서드는 변경했다는 가정하에 작성했습니다. 혹시라도 변경하지 않는다면 기존 메서드를 호출할 때, return 값이 true가 되도록 변경하면 됩니다. 그 외에도 이 메서드에 대한 테스트 부분은 다시 한번 확인하셔야할 것 같아요 🤔) 

```kotlin
@Test
    fun `이미 작성한 지원서가 있습니다`() {
        every { recruitmentRepository.findByIdOrNull(any()) } returns recruitment
        every { applicationFormRepository.notExistsByRecruitmentIdAndUserId(any(), any()) } returns false
        every { applicationFormRepository.save(any<ApplicationForm>()) } returns mockk()
        every { userValidator.validate(any(), any()) } just Runs

        val message = assertThrows<IllegalStateException> {
            applicationFormService.create(
                userId,
                createApplicationFormRequest
            )
        }.message
        assertThat(message).isEqualTo("이미 작성한 지원서가 있습니다.")
    }
```

Reply:
오 이 부분에 대한 테스트가 없었군요..! 꼼꼼한 체크 감사합니다. 해당 테스트는 잘 추가해두었어요 ㅎㅎ
하지만 exists~ 의 메서드를 notExists~으로 바꾼다고 하여 크게 가독성이 좋아질지에 대한 부분은 개인적으로 크게 와닿지 않아서(물론 아주 저의 개인적인 의견이기에..ㅎㅎ) 일단은 남겨두었어요! 
혹시 다른 크루들의 의견이 더 있다면 추가 반영해보도록 하겠습니다!!